libraries/shared/mocks/storage_repository.go

```diff
@@ -37,6 +38,6 @@ func (repository *MockStorageRepository) Create(diffID, headerID int64, metadata
 	return repository.CreateErr
 }
 
-func (*MockStorageRepository) SetDB(db *postgres.DB) {
-	panic("implement me")
+func (repository *MockStorageRepository) SetDB(db *postgres.DB) {
+	repository.db = db
```

Comment:
Maybe this should be a no-op? Thinking that we never want the mock to actually interact with the db